examples/skywalking-tracing/docker-compose.yaml

```diff
@@ -56,7 +56,7 @@ services:
         soft: -1
         hard: -1
   skywalking-oap:
-    image: apache/skywalking-oap-server:8.2.0-es7
+    image: apache/skywalking-oap-server:8.4.0-es7
```

Comment:
Currently we are using cpp2sky v0.1.0 and it supports v8.3.0 data collect protocol. Can we bump this gracefully? https://github.com/SkyAPM/cpp2sky/blob/v0.1.0/bazel/repositories.bzl#L15

Reply:
If you didn't try this with latest version (1.18.0-dev), could you try this with this and check whether it works correctly? It is because 1.17.0 doesn't hire cpp2sky but 1.18.0 will do.

Reply:
Was going to mention the same as @Shikugawa. The versions inside the examples and sandboxes should be kept consistent with the core code. Both Skywalking and cpp2sky should be bumped in https://github.com/envoyproxy/envoy/blob/main/bazel/repository_locations.bzl#L320-L346

- com_github_skyapm_cpp2sky has a newer release than v0.1.1@<2021-01-15 06:00:13> : v0.1.3@<2021-02-03 09:26:16>
- skywalking_data_collect_protocol has a newer release than v8.3.0@<2020-11-29 03:05:22> : v8.4.0@<2021-01-20 15:55:04>

Reply:
@moderation v8.4.0 compatible version of cpp2sky(v0.2.0) will be released soon. If we'd like to keep examples and code completely, we should wait to merge this.

Reply:
@Shikugawa Ok, I see.

Reply:
@Shikugawa Ok, I see.

Reply:
@Shikugawa Ok, I see.